.github/workflows/tests.yml

```diff
@@ -104,56 +102,71 @@ jobs:
           python-version: '3.11'
 
       - name: setup rust
-        uses: actions-rs/toolchain@v1
+        uses: dtolnay/rust-toolchain@master
         with:
-          profile: minimal
           toolchain: stable
-          default: true
           target: x86_64-unknown-linux-gnu
 
       - name: install python dependencies
         run: |
           python -m pip install --upgrade pip
           python -m pip install pycparser
 
-      - name: check that Rust's C API declarations are up to date
+      - name: check that Rust/Python C API declarations are up to date
         run: |
           # bindgen is already installed on GitHub Actions VM
-          ./equistore/scripts/update-core.sh
-          git diff --exit-code
-
-      - name: check that Python's C API declarations are up to date
-        run: |
-          python python/scripts/generate-declarations.py
+          ./scripts/update-declarations.sh
           git diff --exit-code
 
   # check that we can build the Python wheel
   python-build:
     runs-on: ubuntu-20.04
     name: check Python build
-    strategy:
-      matrix:
-        python-version: ['3.7', '3.8', '3.9', '3.10', '3.11']
     steps:
       - uses: actions/checkout@v3
 
       - name: set up Python ${{ matrix.python-version }}
```

Comment:
`${{ matrix.python-version }} `does not exist anymore.

Reply:
what would you think of testing only on the min & max supported Python version? Min to ensure we don't use too modern things, max to ensure we don't use deprecated things

Reply:
Sounds reasonable. I am just wondering why all other packages test the full chain.

Reply:
what do you mean by all other? All other cosmo packages? Then the reason is we added all version because it was easy and did not though too much about it ^^